pkg/writer/sarif.go

```diff
@@ -99,3 +97,13 @@ func getSarifLevel(severity string) string {
 
 	return m[strings.ToLower(severity)]
 }
+
+func getAbsoluteFilePath(resourcePath, filePath string) string {
+	if !filepath.IsAbs(resourcePath) {
+		resourcePath, _ = filepath.Abs(resourcePath)
```

Comment:
Even though error seems unlikely here, I think we should handle error case too !!